chore: website - add cncf community banner to front page

cdrage opened the pull request:
chore: website - add cncf community banner to front page

### What does this PR do?

* Adds a "CNCF" community banner to the front page

### Screenshot / video of UI

<!-- If this PR is changing UI, please include
screenshots or screencasts showing the difference -->

Mockup:
![Screenshot 2025-04-09 at 3 29 59 PM](https://github.com/user-attachments/assets/3355befc-7aa2-413e-82a1-2032556dc1d8)



Implementation:
![Screenshot 2025-04-09 at 3 29 51 PM](https://github.com/user-attachments/assets/0bc9b04e-8e20-43e6-8e8a-7817296a77d3)



### What issues does this PR fix or reference?

<!-- Include any related issues from Podman Desktop
repository (or from another issue tracker). -->

Part of https://github.com/podman-desktop/podman-desktop/issues/11816

### How to test this PR?

<!-- Please explain steps to verify the functionality,
do not forget to provide unit/component tests -->

N/A, view on netlify.

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

cdrage commented:
**Changes from UI/UX mockup:**
cc @MariaLeonova 

* I did not include the boxes since we need higher resolutions of them :( I also think it may be better without?
* The font is our standard `text-base` font / slight gray colour which matches the rest of the website, unlike the mockup which was more white.
* Slight difference in colour with regards to the border, I used instead `sky-blue-500` and `purple-500` which matches the other UI/UX colour components we have.

Firewall reviewed (changes requested):
Hey @cdrage looks like there is still some issues with the background there is a dark background that isn't in the design. 
![image](https://github.com/user-attachments/assets/ff9ef713-e63b-4aae-ad64-84c76b431b28)

cdrage commented:
Great catch!

Updated @Firewall 
![Screenshot 2025-04-14 at 1 31 59 PM](https://github.com/user-attachments/assets/d07c0405-1d01-4079-9582-5aab3da38ee8)

Firewall commented:
Looking a lot closer to the design! Question, can we try giving the box more horizontal space? In the design the box is a lot wide and it looks better IMO. Worth a try. Let me know if it is possible. 

Also @MariaLeonova do we have high resolution versions of the boxes?

cdrage commented:
@Firewall Done!

I've done the following:
- Added more space to the left and right like you mentioned.
- Added the boxes, I got the high resolution boxes from @MariaLeonova 

**Minor changes:**
- The boxes dissapear on smaller screens. This is intentional, else it would be very cluttered / mess up with the text.
- I did NOT include the "white" boxes because with our light theme on the website, it made it dissapear, and when using an inverted coloured version, it did not look good.
- Aligned the boxes better


Large screens:

![Screenshot 2025-04-15 at 10 16 32 AM](https://github.com/user-attachments/assets/23cb890a-b1ed-43b2-9fd3-f7a36e5b2b3f)
![Screenshot 2025-04-15 at 10 16 48 AM](https://github.com/user-attachments/assets/3e5a91a7-ce1d-49af-8e89-96cd607dfdb4)

Medium:
![Screenshot 2025-04-15 at 10 17 10 AM](https://github.com/user-attachments/assets/1a58ca30-2922-465d-8644-be992f09dae5)

Small:

![Screenshot 2025-04-15 at 10 17 21 AM](https://github.com/user-attachments/assets/6cf1efff-056c-45b5-92ef-cddbd11ba057)

Firewall commented:
Looks great, but I dont see any boxes 
![image](https://github.com/user-attachments/assets/5615757b-eb76-4a66-9a0e-e13203857f71)

cdrage commented:
> Looks great, but I dont see any boxes 
> 
> ![image](https://github.com/user-attachments/assets/5615757b-eb76-4a66-9a0e-e13203857f71)
> 
> 

Can you try it locally?

Looks like there is an issue with Netlify not updating. 

The download button in the navbar isn't appearing either despite the code being there.

cdrage commented:
> Boxes are there now! Looks good to me (non code-wise)

Thanks! I've opened up issue https://github.com/podman-desktop/podman-desktop/issues/12238

So we can fix these changes in the future (there is something odd going on with caching and pushing changes..)

MariaLeonova commented:
@cdrage looks good to me!
